### DISCUSSION TITLE
Question/problem: container max-width being overwritten

### DISCUSSION THREAD

**Discussion by wiwi-websolutions** [twbs/bootstrap#31957] [General] [answered]
Hi,

When using v5.0.0-alpha2, nesting containers like this doesn't work anymore...
`<div class="container-fluid">
            <div class="row">
                <div class="container">`

The container `max-width` is being overwritten by:
`.row > * {
    flex-shrink: 0;
    width: 100%;
    max-width: 100%;
    padding-right: calc(var(--bs-gutter-x) / 2);
    padding-left: calc(var(--bs-gutter-x) / 2);
    margin-top: var(--bs-gutter-y);
}`

**Comment by mdo** [comment, marked as answer]
That's correct and intended behavior—containers should not be immediate children of rows.
